### ISSUE TITLE
Exposing Cheating in SQL/SPLdepth: The Impact of Overlapping Training and Test Splits

### ISSUE THREAD

**Issue by ghost** [Lavreniuk/SPIdepth#11]
From [here](https://github.com/Lavreniuk/SPIdepth/issues/2#issuecomment-2343014195), it can be known that this work also use the 70k+ training split. But I need to point out that the 70k training split has a critical issue: there is overlap between the training and test splits!

First, please check 2011_09_30/2011_09_30_drive_0027_sync 656 l, which appears in both the 70k training split and the Eigen test split.

Second, in the traditional Eigen split, there is no overlap in sequences between training and test sets. For example, 2011_10_03_drive_0027_sync is part of the test set and never appears in the training set. However, in the 70k split, this sequence appears 8995 times in the training set!

In other words, SQLdepth is not maintaining the proper separation and is effectively cheating.

Since the SQLdepth author released the training set in April, many people have already identified this error, but for various reasons, they did not report it. As a result, numerous papers have been based on this flawed work. I hope that everyone will be cautious and not continue to trust this repository.

**Comment by Lavreniuk** [comment]
Hi @ghost, thanks for pointing that out—it's helpful to know. I was also curious about why SQLdepth used 70k+ samples for the unsupervised setup but reduced it to 20k+ for fine-tuning. I agree it could affect unsupervised training settings, but considering that the metric depth training maintains a proper split, I think its impact on performance is likely minimal.
